build-go.sh

```diff
@@ -6,6 +6,9 @@ set -eo pipefail
 GOPATH="$(go env GOPATH)"
 export GOPATH
 
+# Always use go modules
+export GO111MODULE=on
+
```

Comment:
Does the build process now rely on Go modules in order to function here?

Reply:
No, but we should not have any more go repos that do not use modules.  Also, `GOPATH` (and therefore also `GO111MODULE`) will not be supported in go1.17.  So, I would like the build to fail if anything still needs `GOPATH` or does not have module support.